module/sheets/WitcherActorSheet.js

```diff
@@ -2185,52 +2126,30 @@ export default class WitcherActorSheet extends ActorSheet {
           break;
       }
 
-      let content = `<h2>${item.name} ${game.i18n.localize("WITCHER.Dialog.attackUse")}: ${attackSkill}</h2> 
-                     <div class="flex">
-                      <label>${game.i18n.localize("WITCHER.Dialog.attackExtra")}: <input type="checkbox" name="isExtraAttack"></label> <br />
-                     </div>
-                     <script>
-                     function myFunction() {
-                       var x = document.getElementById("attackModifiers");
-                       x.style.display = x.style.display === "none" ? "block" : "none";
-                     }
-                     </script>
-                     <label>${game.i18n.localize("WITCHER.Dialog.attackLocation")}: <select name="location">${locationOptions}</select></label> <br />`
-      if (item.system.range) {
-        content += `<label>${game.i18n.localize("WITCHER.Dialog.attackRange")}: <select name="range">${rangeOptions}</select></label> ${item.system.range}<br />`
-      }
-
-      content += `<label>${game.i18n.localize("WITCHER.Dialog.attackStrike")}: <select name="strike">${StrikeOptions}</select></label> <br />
-                  <label>${game.i18n.localize("WITCHER.Dialog.damageType")}: <select name="damageType">${DamageOptions}</select></label> <br />
-                  <label>${game.i18n.localize("WITCHER.Dialog.attackCustom")}: <input type="number" class="small" name="customAtt" value=0></label> <br />
-                  <label>${game.i18n.localize("WITCHER.Dialog.attackModifierse")}: <a onclick="myFunction()"><i class="fas fa-chevron-right"></i></a></label> <br />${AttackModifierOptions}<br />
-                  <h2>${item.name} ${game.i18n.localize("WITCHER.Dialog.attackDamage")}: ${displayDmgFormula}</h2> 
-                  <label>${game.i18n.localize("WITCHER.Dialog.attackCustomDmg")}: <input type="number" class="small" name="customDmg" value=0></label> <br />`;
-                  
-      if (this.actor.type =="character" && isMeleeAttack){ 
-        content += `<label>${game.i18n.localize("WITCHER.Dialog.attackMeleeBonus")}: ${this.actor.system.attackStats.meleeBonus} </label><br />`
-      }
-
+      let ammunitions = ``
+      let noAmmo = 0
+      let ammunitionOption = ``
       if (item.system.usingAmmo){
-        let ammunitions = this.actor.items.filter(function(item) {return item.type=="weapon" &&  item.system.isAmmo});
+        ammunitions= this.actor.items.filter(function(item) {return item.type=="weapon" &&  item.system.isAmmo});
```

Comment:
add space before = and it seems that after  && it has an unessessary space

Reply:
Fixed